storybook/stories/index.js

```diff
@@ -373,13 +373,19 @@ storiesOf('Screens/Login', module)
   .add('ImportMethod Screen', () => <ImportMethod />)
   .add('EnterPrivateKey Screen', () => <EnterPrivateKey />)
   .add('GenerateMnemonic Screen', () => (
-    <GenerateMnemonic 
+    <GenerateMnemonic
       mnemonic='check mnemonic phrase'
       password='check123password'
       privateKey='PrivateKeyCheck'
     />
-  )
-  )
+  ))
+  .add('ConfitmMnemonic Screen', () => (
+    <ConfitmMnemonic
+      mnemonic='test mnemonick for story book'
```

Comment:
Mnemonic phrase must contain exactly 12 words from dictionary. Please use one of generated mnemonic phrases for test purposes.